### DISCUSSION TITLE
Support for masking

### DISCUSSION THREAD

**Discussion by MartijnGP** [hyperion-project/hyperion.ng#1534] [Ideas]
Hi there,

I love using my Hyperion setup for gaming. However unlike with films, games often have a HUD that is also in many cases significantly brighter than the rest of the screen. This leads to my ambilight flaring up permanently for areas of my game that have UI on them.

I'm suggesting a mask-like feature, not unlike those found in for example Photoshop layers, just a black and white image that defines which areas of the input to ignore, preferably switchable through the remote or REST API.

Even better would be to have the masked pixels take an average of nearby non-masked areas, but I can imagine that's significantly harder.

I can do a little bit of coding but video manipulation is out of my league. Maybe someone more skilled than I am would find this useful as well.

**Comment by Lord-Grey** [comment]
Hi @MartijnGP 

> that defines which areas of the input to ignore

That would  mean that for those ares the LEDs would stay black.
Currently, the LED-Layout already provide the ability to Blacklist LEDs and their mapping area via the "Blacklist" feature.
Maybe you give it a try.

## Sample
![image](https://user-images.githubusercontent.com/48840279/204498192-ade3f9f0-bdb6-4b1e-a893-e00e326d0ffa.png)

> preferably switchable through the remote or REST API.

Unfortunately, it is currently not possible to dynamically change the blacklisted LED mappings.
